### ISSUE TITLE
Camera is not cover full screen of the device. 

### ISSUE THREAD

**Issue by Yesha052** [Firekast-io/firekast-demo-android#9]
![fbscreen](https://user-images.githubusercontent.com/48985106/63690851-57945a80-c82b-11e9-9f6d-e00a9dba4768.png)

Please see the sides of the screen.

**Comment by francoisrouault** [comment]
We decided to display the camera center inside its fragment. We have to also consider a center crop scale type, it will be available in the next releases.

**Comment by Yesha052** [comment]
Hi @francoisrouault , 
Can you please provide the dates for next release?  or any other alternative to fix this?

**Comment by francoisrouault** [comment]
Hi @Yesha052,

Sorry for the delay. We have just released a new version 1.7.3 (and updated demo app subsequently) offering a center crop scale type for the camera view!

It is easy to setup: use the `FKCameraFragment.Builder` to build your `FKCameraFragment` instance. While building, you can chose the scale type to apply.
```
FKCameraFragment cameraFragment = new FKCameraFragment.Builder()
   .setCameraScaleType(FKCamera.ScaleType.CENTER_CROP)
   .build();
```
